content/_data/changelogs/weekly.yml

```diff
@@ -11293,6 +11293,47 @@
   # pull: 5535 (PR title: Bump commons-io from 2.8.0 to 2.9.0)
   # pull: 5546 (PR title: Bump checkstyle from 8.42 to 8.43)
 
+- version: '2.297'
+  date: 2021-06-08
+  changes:
+    - type: bug
```

Comment:
can we update to include https://github.com/jenkinsci/jenkins/pull/5548 as well?

Reply:
Sure! Actually I tried to run the changelog-generator to create the template but it is checking changes from "commit f34..." (tag: jenkins-2.297) to "commit 3a1...", and as you can see, jenkinsci/jenkins#5548 is at "commit cb9..." which is not coming under this range. This is a problem I haven't come across before, so not sure how to change the range here.

![issue_changelog-2 297](https://user-images.githubusercontent.com/43273420/121229017-14de2e00-c8ab-11eb-8282-f9bf685ea8ce.png)

Reply:
However I did try to hardcode it in the meantime, 

```
- type: bug
      pull: 5548
      issue: 65195
      authors:
      - Wadeck
      references:
        - pull: 5548
        - issue: 65195
        - issue: 64347
      message: |-
        Fix incorrect process termination issues when running on macOS.
        Add check to prevent out of bounds memory access on macOS.
```

and it looks like this:
![image](https://user-images.githubusercontent.com/43273420/121232371-29bcc080-c8af-11eb-8f26-a05e73b79252.png)

Reply:
The output you showed looks like it is for the current release, i.e. what will become 2.298, see message on top.

Reply:
Yes, so pls correct me if I'm wrong, so then I need to check changes from 2.296 till the latest PR.
To do that, I tried to do echo '2.296' to the latestCore.txt file, but still, it is checking changes from 2.297

Reply:
See https://github.com/jenkinsci/core-changelog-generator/blob/master/README.md#usage-in-docker for usage instructions.

Reply:
Thank you for your help!!